lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -656,6 +656,15 @@ def write(chunk)
                          "server, dropping #{entries_count} log message(s)",
                          error: error.to_s, error_code: error.code.to_s
             end
+
+          # Got an unexpected error (not Google::Gax::GaxError) from the
+          # google-cloud-logging lib.
+          rescue StandardError => error
+            increment_failed_requests_count(error.code)
```

Comment:
A `StandardError` won't have a `code` field or method, would it?

Reply:
Changed it to `GRPC::Core::StatusCodes::UNKNOWN` instead.